tui/helpers.go

```diff
@@ -0,0 +1,111 @@
+package tui
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+	"time"
+
+	"github.com/augmentable-dev/askgit/pkg/gitqlite"
+	"github.com/go-git/go-git/v5"
+
+	"github.com/jroimartin/gocui"
+)
+
+//Gets the repo from the provided path(can be local ssh or http(s))
+func GetRepo(remote string) (string, error) {
```

Comment:
this seems duplicative of code that exists in `root.go` - if you move the call to `RunGUI(...)` lower down in `root.go` and pass in the "final" path to the repo on disk (if it get's cloned to a tmp dir) - rather than a string that may be a path on disk or a remote repo reference, you can probably remove this whole func

Reply:
If I do that I am unable to give display the Repo's origin in the Info box. A very small thing but if it is better to have the code be more simplified that can be arranged.